src/clipboard_handlers/borders_clipboard.ts

```diff
@@ -26,6 +26,9 @@ export class BorderClipboardHandler extends AbstractCellClipboardHandler<
     const borders: (Border | null)[][] = [];
 
     for (const row of rowsIndexes) {
+      if (!isCutOperation && this.getters.isRowFiltered(sheetId, row)) {
+        continue;
+      }
```

Comment:
idea: remove filtered rows directly from from rowsIndexes, that way we do'nt ahve to repeat this every clipbboard hanldelr